runservices.sh

```diff
@@ -1,11 +1,15 @@
-#!/bin/bash
-pipenv run pip install --upgrade pip
-pipenv install 
+#!/usr/bin/env sh
```

Comment:
There are some things that don't always run in sh but run in bash. I'm not comfortable changing that.

Reply:
I may include [values] in the scripts at some point

Reply:
> There are some things that don't always run in sh but run in bash. I'm not comfortable changing that.

I don't know the reason why you aren't comfortable, but Alpine comes with `sh` because of BusyBox.